extensions/spatial.sigmf-ext.md

```diff
@@ -203,15 +203,20 @@ object:
 |----|--------|----|-----|-----------|
 |`signal_azimuth`|false|double|degrees|Azimuth in degrees east of true north associated with the specific annotation.|
 |`signal_bearing`|false|[bearing](spatial.sigmf-ext.md#01-the-bearing-object)|N/A|Bearing associated with the specific annotation.|
+| `geolocation`  | false    | GeoJSON `point` Object | N/A | The location (either known or estimated) of the transmitter corresponding to this annotation |
```

Comment:
lets call this `signal_geolocation` or `emitter_geolocation` to avoid confusion with the [Annotations Scope `latitude` and `longitude` fields](https://github.com/sigmf/SigMF/blob/sigmf-v1.x/sigmf-spec.md#annotations-array).

Reply:
Yes, it is indeed confusing. Originally these fields were designed to hold the _system_ location and were annotations-scoped to allow changing position. This predates me a bit, but I think everyone agrees it was not the ideal choice.

These have been deprecated since 0.9 and will be formally removed in 2.0